scripts/update_versions.py

```diff
@@ -2,20 +2,27 @@
 
 from playwright.sync_api import sync_playwright
 
-with sync_playwright() as p:
-    r = open("README.md", "r")
-    text = r.read()
-    for browser_type in [p.chromium, p.firefox, p.webkit]:
-        rx = re.compile(
-            r"<!-- GEN:" + browser_type.name + r"-version -->([^<]+)<!-- GEN:stop -->"
-        )
-        browser = browser_type.launch()
-        text = rx.sub(
-            f"<!-- GEN:{browser_type.name}-version -->{browser.version}<!-- GEN:stop -->",
-            text,
-        )
-        browser.close()
 
-    w = open("README.md", "w")
-    w.write(text)
-    w.close()
+def main() -> None:
+    with sync_playwright() as p:
+        with open("README.md", "r") as r:
```

Comment:
using pathlib with read_text and write_text is probably cleaner then it's less nested and redundated code.

Reply:
I was just about to do it 🚀